contents/docs/hog/index.mdx

```diff
@@ -284,6 +286,47 @@ Hog's standard library includes the following functions and will expand. To see
 - `sha256Hex(arg: string): string`
 - `sha256HmacChainHex(arg: string[]): string`
 
+## Why Hog?
+
+Truth be told, we didn't set out to build our own programming language. We actually wanted to build the **HogVM**: a way to run HogQL expressions synchronously within our NodeJS ingestion pipeline. However, things escalated quickly.
+
+### HogVM
+
+The HogVM is a stack-based bytecode virtual machine (VM). It was built to power event filtering in our real-time ingestion flow.
```

Comment:
I think, most of the time, when you use "ingestion" here you mean "delivery" (or, to customers at least, that's what will be clearer) - "It was built to power event filtering in our real-time delivery system" reads clearer to me.